### ISSUE TITLE
arabic translation

### ISSUE THREAD

**Issue by elmehdiboubetana** [Stremio/stremio-translations#171]
Hi all , 

i'm working on an arabic ( ar-AR.json )  translation for stremio , but when i come to the test , my .json file puted in the main directory with stremio , with this command nothing happend : Stremio.exe . --translation=ar-AR.json 

Please Advise

**Comment by Ivshti** [comment]
try providing the full path to the .json file

**Comment by elmehdiboubetana** [comment]
Yes i provide the full path of the file but still have nothing please see the screenshot .

![capture](https://cloud.githubusercontent.com/assets/4965993/18615597/52633c38-7da2-11e6-978c-945031fcad5d.JPG)

**Comment by Ivshti** [comment]
perhaps there is an error in the json file? can you run it by an online json validator? or simply try to require() it with node.js

**Comment by xeanhort** [comment]
I have the same problem here. I'm executing it like:

`C:\Users\XXX\AppData\Local\Programs\LNV\Stremio-4>stremio.exe . --translation=D:\stremio-translations\ca-CA.json`

I don't think is a file issue because if I try some other valid file it doesn't work either.



> try to require() it with node.js
Where should I put this require?

**Comment by dexter21767** [comment]
your code should be like this 
Stremio.exe . --translation="ar-AR.json"
